docs/reference/cli.md

```diff
@@ -542,27 +543,33 @@ Create an access key
 Create an access key for a user or a connector.
 
 ```bash
-infra keys add USER|connector [flags]
+infra keys add [flags]
 ```
 
 ### Examples
 
 ```bash
 
 # Create an access key named 'example-key' for a user that expires in 12 hours
-$ infra keys add user@example.com --ttl=12h --name example-key
+$ infra keys add --ttl=12h --name example-key
 
 # Create an access key to add a Kubernetes connection to Infra
-$ infra keys add connector
+$ infra keys add --connector
```

Comment:
We will also have to update `kubernetes.md` and `NOTES.txt` with this new format